server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/MultipleDatabaseChangeRuntimeFlowableTask.java

```diff
@@ -211,10 +209,10 @@ protected void onFailure(Long taskId, TaskService taskService) {
                     this.batchId == null ? null : this.batchId + 1);
             updateFlowInstanceStatus(FlowStatus.EXECUTION_FAILED);
             taskService.fail(taskId, 100, generateResult());
-            super.onFailure(taskId, taskService);
         } finally {
             MultipleDatabaseChangeTraceContextHolder.clear();
         }
+        super.onFailure(taskId, taskService);
```

Comment:
The location of this code must be changed？it is recommended to keep the original location

Reply:
There is no impact on the code logic compared to before.